src/LEGO.AsyncAPI/Models/Bindings/Kafka/KafkaChannelBinding.cs

```diff
@@ -15,27 +15,27 @@ public class KafkaChannelBinding : IChannelBinding
         /// <summary>
         /// Kafka topic name if different from channel name.
         /// </summary>
-        public string Topic { get; set; }
+        public string? Topic { get; set; }
 
         /// <summary>
         /// Number of partitions configured on this topic (useful to know how many parallel consumers you may run).
         /// </summary>
-        public int Partitions { get; set; }
+        public int? Partitions { get; set; }
 
         /// <summary>
         /// Number of replicas configured on this topic.
         /// </summary>
-        public int Replicas { get; set; }
+        public int? Replicas { get; set; }
 
         /// <summary>
         /// Topic configuration properties that are relevant for the API.
         /// </summary>
-        public TopicConfigurationObject TopicConfiguration { get; set; }
+        public TopicConfigurationObject? TopicConfiguration { get; set; }
```

Comment:
I believe this is nullable as well, by default. (WriteOptional - will simply null it)